### PR TITLE
[CI/Build] Add Model Tests for PixtralHF

### PR DESCRIPTION
**Pull request by mgoin**
Adds a simple vLLM vs HF test for PixtralHF models

Tested locally with
```
pytest tests/models/decoder_only/vision_language/test_models.py -k "pixtral" -s -v
...
============================= 8 passed, 109 deselected, 77 warnings in 657.77s (0:10:57) ==============================
```

### CODE REVIEW

**Comment by ywang96** [comment]
Hey @mgoin can you hold this PR until we merge https://github.com/vllm-project/vllm/pull/9372?

**Comment by mgoin** [comment]
Ah okay @ywang96, sure it can wait

**Review by DarkLight1337 — APPROVED** [review]

File: tests/models/decoder_only/vision_language/test_models.py
Comment (DarkLight1337) [inline tests/models/decoder_only/vision_language/test_models.py:297]:
```suggestion
        prompt_formatter=lambda img_prompt: f"<s>[INST]\n{img_prompt}[/INST]",
```

Comment (mgoin) [reply tests/models/decoder_only/vision_language/test_models.py:297]: From the HF example, the newline is actually supposed to be between the prompt and the images i.e.
```
PROMPT = "<s>[INST]Describe the images.\n[IMG][IMG][IMG][IMG][/INST]"
```

https://github.com/vllm-project/vllm/blob/77f7ef29088fef854421239e7c41df6b11bc4b5b/examples/offline_inference_vision_language.py#L295

I wasn't sure how to do this with this structure, but it wasn't important to get matching vLLM vs HF

Comment (DarkLight1337) [reply tests/models/decoder_only/vision_language/test_models.py:297]: Oh, I see now. You can use the `img_idx_to_prompt` argument for this.

Comment (mgoin) [reply tests/models/decoder_only/vision_language/test_models.py:297]: Yeah I did use `img_idx_to_prompt=lambda idx: "[IMG]",` and that worked thankfully. Luckily the newline isn't crucial for correctness, so that feature can wait

**Pull request merged**
